writer/app.js

```diff
@@ -1,95 +1,98 @@
-    }, 0);
-    const itemNameTextNode = document.createTextNode(item);
-    const counterTextNode = document.createTextNode(`(${highestCounter + 1})`);
-    const spaceTextNode = document.createTextNode(' ');
-
-    const listDiv = document.createElement('div');
-    listDiv.classList.add('listItem');
-    const list = document.createElement('li');
-    list.dataset.count = highestCounter + 1;
-    console.log(`the counter for ${item} is at ${list.dataset.count}`);
-    
-    list.appendChild(itemNameTextNode);
-    list.appendChild(spaceTextNode);
-    list.appendChild(counterTextNode);
-    listDiv.appendChild(list);
-
-    const deleteButton = document.createElement('button');
-    deleteButton.textContent = 'Delete';
-    listDiv.id = input;
-    listDiv.appendChild(deleteButton);
-    outputArea.appendChild(listDiv);
-    
-    deleteButton.addEventListener('click', function () {
-        deleteButtonFunction(listDiv, outputArea, item);
-    });
+  }, 0);
+
+  const itemNameTextNode = document.createTextNode(item);
+  const counterTextNode = document.createTextNode(`(${highestCounter + 1})`);
+  const spaceTextNode = document.createTextNode(' ');
+
+  const listDiv = document.createElement('div');
+  listDiv.classList.add('listItem');
+  const list = document.createElement('li');
+  list.dataset.count = highestCounter + 1;
+
+  list.appendChild(itemNameTextNode);
+  list.appendChild(spaceTextNode);
+  list.appendChild(counterTextNode);
+  listDiv.appendChild(list);
+
+  const deleteButton = document.createElement('button');
+  deleteButton.textContent = 'Delete';
+  deleteButton.setAttribute("class","btn btn-success")
+  deleteButton.setAttribute("id","delete-btn")
+
+  listDiv.id = input;
+  listDiv.appendChild(deleteButton);
+  outputArea.appendChild(listDiv);
+
+  deleteButton.addEventListener('click', function () {
+      deleteButtonFunction(listDiv, outputArea, item);
+  });
 }
 
-    function deleteButtonFunction(listDiv, outputArea, item) {
-        const currentCounter = parseInt(listDiv.firstChild.dataset.count) || 0;
-        console.log("Delete button is clicked");
-
-        if (currentCounter > 1) {
-            // Decrease the item counter by one
-            listDiv.firstChild.dataset.count = currentCounter - 1;
-            console.log(`Counter for ${item} decreased to ${listDiv.firstChild.dataset.count}`);
-
-            // Update the text content to reflect the new counter
-            listDiv.firstChild.textContent = `${item} (${currentCounter - 1})`;
-          } else { 
-            outputArea.removeChild(listDiv);
-            console.log('Item successfully deleted');
-          }
-            }
-
-
-
-    
-  // Function to send the shopping list to the server
-    function sendShoppingList() {
-        const outputArea = document.getElementById('outputArea');
-        const shoppingListItems = Array.from(outputArea.children)
-            .map(item => item.firstChild.textContent.trim());
-
```

Comment:
why are you removing the send-list API

Reply:
didnt meant to ! probably deleted it by mistake